pkg/util/cmdutil/console.go

```diff
@@ -93,3 +94,110 @@ func RemoveTralingNewline(s string) string {
 
 	return s
 }
+
+// TableRow is a row in a table we want to print.  It can be a series of a columns, followed
+// by an additional line of information.
+type TableRow struct {
+	Columns        []string
+	AdditionalInfo string
+}
+
+// PrintTable prints a grid of rows and columns.  Width of columns is automatically
+// determined by the max length of the items in each column.
+func PrintTable(table []TableRow) {
```

Comment:
This is definitely a nice improvement!

I suppose we could have taken a `headers` array, maybe of just `[]string`, and did the UPPER-CASING, so that the callers don't need to worry about that aspect of formatting too.

Either way, definitely happy you made this improvement.

Reply:
Have changed it to break out the headers from the rows.